### ISSUE TITLE
Wich LoRa library is used?

### ISSUE THREAD

**Issue by ClimberWue** [LilyGO/TTGO-LORA32-V2.0#23]
In your code I found
#include <lora.h>
Do you have a link to the proper source or library?

**Comment by morganrallen** [comment]
It can be downloaded through the Arduino Library Manager.
From the Arduino IDE...

Sketch > Include Library > Manage Libraries
Type LoRa into the search and find the one by "Sandeep Mistry"

The official repo for this library is at https://github.com/sandeepmistry/arduino-LoRa/

**Comment by TrullT** [comment]
Thanks  - saved me a lot of time. It would be great if this Library metadata was recorded in a comment thusly:
#include <LoRa.h> // https://github.com/sandeepmistry/arduino-LoRa/ or Arduino Library Manager

**Comment by morganrallen** [comment]
I agree and hope to work with LilyGO in the future to make these sort of things more clear.

**Comment by jbrepogmailcom** [comment]
Hello, what library do you recommend for LoRaWAN? Will the ibm-lmic library work without pin redefinition?
